website/docs/docs/explore/data-tile.md

```diff
@@ -36,9 +36,9 @@ Data health tiles rely on [exposures](/docs/build/exposures) to surface data hea
   -  If using manual exposures, they must be explicitly defined in your YAML files.
   - If using automatic downstream exposures, ensure your BI tool is [configured](/docs/cloud-integrations/downstream-exposures-tableau) with <Constant name="cloud" />.
 - You have [source freshness](/docs/deploy/source-freshness) enabled in the job that generates this exposure.
-- The exposure used for the  data health tile must have the [`type` property](/docs/build/exposures#available-properties) set to `dashboard`. Otherwise, you won't be able to view the **Embed data health tile in your dashboard** dropdown in dbt Explorer.
+- The exposure used for the  data health tile must have the [`type` property](/docs/build/exposures#available-properties) set to `dashboard`. Otherwise, you won't be able to view the **Embed data health tile in your dashboard** dropdown in dbt Catalog.
```

Comment:
```suggestion
- The exposure used for the data health tile must have the [`type` property](/docs/build/exposures#available-properties) set to `dashboard`. Otherwise, you won't be able to view the **Embed data health tile in your dashboard** dropdown in <Constant name="explorer" />.
```